PEPit/examples/continuous_time_models/accelerated_gradient_flow_convex.py

```diff
@@ -0,0 +1,123 @@
+import cvxpy as cp
+from PEPit import PEP
+from PEPit.functions.convex_function import ConvexFunction
+
+
+def wc_accelerated_gradient_flow_convex(t, verbose=1):
+    """
+    Consider the convex minimization problem
+
+    .. math:: f_\\star \\triangleq \\min_x f(x),
+
+    where :math:`f` is convex.
+
+    This code computes a worst-case guarantee for an **accelerated gradient** flow.
+    That is, it verifies the inequality
+
+    .. math:: \\frac{d}{dt}\\mathcal{V}(X_t, t) \\leqslant 0 ,
+
+    is valid, where :math:`\\mathcal{V}(X_t, t) = t^2(f(X_t) - f(x_\\star)) + 2 \\|(X_t - x_\star) + \\frac{t}{2}\\frac{d}{dt}X_t \\|^2`,
+    :math:`X_t` is the output of an **accelerated gradient** flow, and where :math:`x_\\star` is the minimizer of :math:`f`.
+    In short, for given values of :math:`t`, it verifies :math:`\\frac{d}{dt}\\mathcal{V}(X_t, t) \\leqslant 0`.
+
+    **Algorithm**:
+    For :math:`t \\geqslant 0`,
+
+                .. math:: \\frac{d^2}{dt^2}X_t + \\frac{3}{t}\\frac{d}{dt}X_t + \\nabla f(X_t) = 0,
+
+    with :math:`X_{0}:= x_0 \\in \\mathbf{R}^d`.
+
+    **Theoretical guarantee**:
+
+        The following **tight** guarantee can be verified in [1, Section 2]:
+
+        .. math:: \\frac{d}{dt}\\mathcal{V}(X_t, t) \\leqslant 0.
+
+        After integrating between :math:`0` and :math:`t`,
+
+        .. math:: f(X_t) - f_\\star \\leqslant \\frac{2}{t^2}\\|x_0 - x_\\star\\|^2.
+
+    **References**:
+
+    `[1] W. Su, S. Boyd, E. J. Candès (2016).
```

Comment:
add ref to your own work for the description of the PEP formulation for this setup?